packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -2436,4 +2463,17 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
       // },
     },
   };
+
+  private _estimateGasWithMargin = async (
+    txData: ethers.PopulatedTransaction,
+    porcentualIncrease: number = GAS_LIMIT_INCREASE_PERCENTAGE
+  ) => {
+    const gasEstimation = await this.pkpNftContract.write.provider.estimateGas(
+      txData
+    );
+    const adjustedGasLimit = gasEstimation
+      .mul(100 + porcentualIncrease)
+      .div(100);
+    return adjustedGasLimit;
+  };
```

Comment:
Can we change this to use english (percentage) for consistency w/ the constant? :)

```suggestion
  private _estimateGasWithMargin = async (
    txData: ethers.PopulatedTransaction,
    percentageIncrease: number = GAS_LIMIT_INCREASE_PERCENTAGE
  ) => {
    const gasEstimation = await this.pkpNftContract.write.provider.estimateGas(
      txData
    );
    const adjustedGasLimit = gasEstimation
      .mul(100 + percentageIncrease)
      .div(100);
    return adjustedGasLimit;
  };
```